cpp/tests/strings/floats_tests.cpp

```diff
@@ -118,38 +117,41 @@ TEST_F(StringsConvertTest, FromFloats32)
     h_expected.end(),
     thrust::make_transform_iterator(h_expected.begin(), [](auto str) { return str != nullptr; }));
 
-  CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(*results, expected, verbosity);
+  CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(*results, expected);
 }
 
 TEST_F(StringsConvertTest, ToFloats64)
 {
   // clang-format off
   std::vector<const char*> h_strings{
-    "1234",   nullptr,    "-876",     "543.2",         "-0.12",   ".25",
+    "1234",   "",         "-876",     "543.2",         "-0.12",   ".25",
```

Comment:
Should this be removed altogether as we already have `""` in the vector? Also curious why we changed `nullptr` to `""` 🤔

Reply:
Previous code used nullptr to identify null entries. This change opted to use a validity iterator instead since it is more easily reused. 
```
auto validity = cudf::test::iterators::null_at(1); // row 1 is a null entry
```
A null row entry is a strings column must be empty otherwise it creates an unsanitized null.